src/feature_types/feature.js

```diff
@@ -55,4 +50,4 @@ Feature.prototype.internal = function(mode) {
   };
 };
 
-module.exports = Feature;
+export default Feature;
```

Comment:
Curious: why switch to using ES2015 exports but still CommonJS `require` instead of `import`?